_posts/2020-10-07-solidity-0.7.3-release-announcement.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.7.3 Release Announcement'
+date: '2020-10-07'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.7.3](https://github.com/ethereum/solidity/releases/tag/v0.7.3) fixes a bug in the dynamic-array cleanup. To learn more about the bug and check if your contract is vulnerable please read this [post]() with all details about dynamic array cleanup bug. Additionally, v0.7.3 adds the option to stop compilation after the parsing stage using ``solc --stop-after parsing``.
+
+## Important Bugfixes
```

Comment:
Should we repeat this below in the "full changelog"? Should we also re-link the bug blog post here?

Reply:
I'll added it to the full change log and added the bug post link